.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
These are my best guess on suggested extensions. Vetur might be better at this point, though I did make a few changes so that Volar would work (which seems to be the preferred extension now).